### PR TITLE
Image server212

### PR DESCRIPTION
**Pull request by aidanwilliams09**
Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [x] Reference the issue this is meant to fix or describe it if none exists.
- [x] Full description of changes made.

## Pull Request Checklist

- [x] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [x] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [x] The Docker image builds
- [x] The container runs correctly when sent inputs with the changes
- [x] No models or other large files are part of these commits
- [x] A description of the tests already run as part of this PR is present

### CODE REVIEW

**Comment by aidanwilliams09** [comment]
This pull request expands the maps module to allow for google places API calls in order to resolve unknown lat/long.

Additionally, this contains some basic refactors to reduce redundancy of validating data/schemas.

File: docker-compose.yml [hunk 1]
```diff
@@ -49,6 +49,8 @@ services:
             context: ./preprocessors/autour
             dockerfile: Dockerfile
         image: "autour:latest"
+        env_file:
+            - ./config/maps.env
```
Comment (JRegimbal) [inline docker-compose.yml:53]: Does this contain an API key?

File: docker-compose.yml [hunk 2]
```diff
@@ -163,17 +165,6 @@ services:
             ca.mcgill.a11y.image.handler: enable
         volumes:
             - sc-store:/tmp/sc-store
-    generic-tts-handler:
```
Comment (JRegimbal) [inline docker-compose.yml:166]: Why did you delete this?

**Review by JRegimbal — COMMENTED** [review]
Some questions, but I don't review PRs for preprocessors, @jaydeepsingh25 does. Adding him here...

File: docker-compose.yml (hunk 1 above)
Comment (aidanwilliams09) [reply docker-compose.yml:53]: Yes, it contains Jeremy's Google API key, obviously this file was not added to the repo.

File: docker-compose.yml (hunk 2 above)
Comment (aidanwilliams09) [reply docker-compose.yml:166]: If you look at the lines above, there is a carbon copy of the ```generic-tts-handler```. I was not able to build the container when there were two processes named the same thing. If there is some purpose for there being two then I can add it back in.

Comment (JRegimbal) [reply docker-compose.yml:166]: Oh no in that case don't worry. It only shows me the changes by default so it looked like you were deleting a whole thing...weird because we aren't having this problem in main as far as I can tell!

Comment (aidanwilliams09) [reply docker-compose.yml:166]: See lines 157-167. 

I don't know how to link it because it's not part of the diff so it isn't giving me an option to comment on the lines.

**Comment by aidanwilliams09** [comment]
Tested on unicorn and received back valid responses for query with lat/long and without. Additionally, tested bad query and received error codes and messages.

**Review by jaydeepsingh25 — APPROVED** [review]
